public/src/components/channelManagement/epicTests/epicVariantPreview.tsx

```diff
@@ -2,16 +2,102 @@ import React from 'react';
 import { Theme, makeStyles } from '@material-ui/core';
 import { EpicVariant } from './epicTestsForm';
 
-import { EpicModuleName } from '../helpers/shared';
+import { EpicModuleName, TickerSettings } from '../helpers/shared';
 import { useModule } from '../../../hooks/useModule';
 
+// Article count TS defs
 export interface ArticleCounts {
-  for52Weeks: number; // The user's total article view count, which currently goes back as far as 52 weeks
-  forTargetedWeeks: number; // The user's article view count for the configured periodInWeeks
+  // The user's total article view count, which currently goes back as far as 52 weeks
+  for52Weeks: number;
+  // The user's article view count for the configured periodInWeeks
+  forTargetedWeeks: number;
+}
+
+// Choice card TS defs and object generation
+interface ChoiceCardAmounts {
+  amounts: number[];
+  defaultAmount: number;
+}
+
+interface ChoiceCardVariant {
+  name: string;
+  amounts: {
+    [index: string]: ChoiceCardAmounts;
+  };
+}
+
+interface RegionalChoiceCard {
```

Comment:
Can we use the same model as SDC? https://github.com/guardian/support-dotcom-components/blob/main/packages/shared/src/types/abTests/epic.ts#L77
Ideally these types would be published as a library, but that's for another day...

Reply:
Done